cmd/wit-bindgen-go/cmd/generate/generate.go

```diff
@@ -189,3 +196,28 @@ func writeGoPackages(packages []*gen.Package, cfg *config) error {
 	}
 	return nil
 }
+
+func writeWITPackage(res *wit.Resolve, cfg *config) error {
+	witDir := filepath.Join(cfg.out, "wit")
+	if err := os.MkdirAll(witDir, cfg.outPerm); err != nil {
+		return err
+	}
+	witFilePath := filepath.Join(witDir, "webassembly.wit.wasm")
+	fmt.Fprintf(os.Stderr, "Generated WIT file: %s\n", witFilePath)
+
+	wasmTools, err := exec.LookPath("wasm-tools")
+	if err != nil {
+		return err
+	}
+
+	var stderr bytes.Buffer
+	wasmCmd := exec.Command(wasmTools, "component", "wit", "--wasm", "--all-features", "--output", witFilePath)
```

Comment:
Per discussion from Luke and others at Plumbers Summit, I think we should emit WIT, and not wasm/assembly here.

Reply:
yup, just changed it